views/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- page title -->
+    <title>About</title>
+    <!-- link to ABOUT css file -->
+    <link rel="stylesheet" href="/public/styles/stylesAbout.css" />
```

Comment:
You need to remove the initial `/public` part of the path to your file. If you test it on localhost, you'd see that the files are not linked together.